typedb/concept/thing/relation.py

```diff
@@ -18,78 +18,55 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-from typing import Iterator
 
-import typedb_protocol.common.concept_pb2 as concept_proto
+from __future__ import annotations
+from typing import Iterator, Any, TYPE_CHECKING
 
-from typedb.api.concept.thing.relation import Relation, RemoteRelation
-from typedb.api.concept.type.relation_type import RelationType
+from typedb.api.concept.thing.relation import Relation
 from typedb.api.concept.type.role_type import RoleType
-from typedb.common.rpc.request_builder import relation_add_player_req, relation_remove_player_req, \
-    relation_get_players_req, relation_get_players_by_role_type_req, relation_get_relating_req
-from typedb.concept.proto import concept_proto_builder, concept_proto_reader
-from typedb.concept.thing.thing import _Thing, _RemoteThing
+from typedb.common.streamer import Streamer
+from typedb.concept.concept_factory import relation_type_of, role_type_of, thing_of
+from typedb.concept.thing.thing import _Thing
 from typedb.concept.type.role_type import _RoleType
```

Comment:
We don't need this import here, because we use `role_type_of()`. But without direct importing of `role_type.py` Bazel think that we don't need it and removes this file. One should investigate this behaviour.